pkg/yurttunnel/server/serveraddr/addr_test.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2020 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package serveraddr
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestGetDefaultDomainsForSvcInputParamEmptyChar(t *testing.T) {
+	domains := getDefaultDomainsForSvc("", "")
+	if len(domains) == 0 {
+		t.Log("domains len is zero")
+	}
```

Comment:
if length of domains is not 0, the case is also passed. so it is better to use `if len(domains) != 0` verification and then `t.Errorf()` to output error message.

Reply:
OK,I will modify it later.

Reply:
I modified it